nix/README.md

```diff
@@ -0,0 +1,63 @@
+# Nix-based kubernetes test environment
+
+All operations are based on having the `nix` package manager available. It can be easily be installed using the quickstart instructions found at https://nixos.org/download.html#nix-quick-install, which are just
+
+```console
+$ curl -L https://nixos.org/nix/install | sh
+```
+
+## Building a docker image of a node
+
+To build your current source into a container image, enter a nix-shell in the same folder as this `README.md`, then run `build-node.sh`.
+
+```console
+$ nix-shell
```

Comment:
Probably should have `nix-shell --pure` here, although see my comment below.

Reply:
I usually don't use `--pure` for dev shells, i.e. when not testing packaging, but using nix-shell to make tools more available, since it removes my personal development tools from the path.